packages/core/utils/src/config.js

```diff
@@ -87,7 +88,7 @@ export async function loadConfig(
       if (parse === false) {
         config = configContent;
       } else {
-        let parse = getParser(extname);
+        let parse = opts?.parser ?? getParser(extname);
```

Comment:
That isn't part of the `configCache` cache key. But I also can't think of a case where you would want to parse a single file with 2 different parsers.

Reply:
Yeah, let's hope not 😉